ouroboros-consensus-test/test-storage/Test/Ouroboros/Storage/ChainDB/FollowerPromptness.hs

```diff
@@ -0,0 +1,214 @@
+{-# LANGUAGE BlockArguments      #-}
+{-# LANGUAGE DerivingStrategies  #-}
+{-# LANGUAGE LambdaCase          #-}
+{-# LANGUAGE NamedFieldPuns      #-}
+{-# LANGUAGE RecordWildCards     #-}
```

Comment:
I prefer `NamedFieldPuns` over `RecordWildCards` in patterns, unless it's a big verbosity increase. The main use in this module saves four fields in the pattern, two of which are quite long names---it's not obviously on either side of my personal threshold :)

Reply:
As I added more fields to the `Outcome` record on your suggestion, we now make even more use of `RecordWildCards`, so I kept it for now :smile: